package.json

```diff
@@ -33,8 +33,9 @@
   },
   "dependencies": {
     "eventemitter3": "1.1.1",
-    "remote-pay-cloud-api": "1.4.1",
-    "typescript": "2.4.1"
+    "remote-pay-cloud-api": "1.4.2",
+    "typescript": "2.4.1",
+    "react-router-dom": "latest"
```

Comment:
Why is react-router-dom a dependency?  We don't want this as part of our remote-pay-cloud distro.

Reply:
This package.json is under the pidgin-test-app sub project.